examples/image-builder.yaml

```diff
@@ -75,6 +75,7 @@ deployment_groups:
       instance_image:
         family: $(vars.new_image_family)
         project: $(vars.project_id)
+      bandwidth_tier: gvnic_enabled
```

Comment:
nit: I would put this below the `instance_image_custom` setting to keep the image settings next to each other.

Reply:
Done.